From 692f0f31df7285559e15e754e679af9fe351a7d9 Mon Sep 17 00:00:00 2001
From: Phong Tran <tranmanphong@gmail.com>
Date: Tue, 19 Jul 2016 07:00:45 +0700
Subject: [PATCH 2/7] use external 3rd party source

set the flag for not build 3rd party source code
apr, apr-utils, log4cxx, dbus ..
Thoes packages are built by oe.

Signed-off-by: Phong Tran <tranmanphong@gmail.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b81e321..791b052 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -395,6 +395,8 @@ set(3RD_PARTY_BINARY_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/src/3rd_party)
 set (install-3rd_party_logger_var "")
 set (install-3rd_party_dbus_var "")
 
+set (NO_REBUILD_3RD_PARTY ON)
+
 if(NO_REBUILD_3RD_PARTY)
   set(NO_REBUILD_3RD_PARTY_LOGGER ON)
   set(NO_REBUILD_3RD_PARTY_DBUS ON)
-- 
1.9.1

